client/src/App.jsx

```diff
@@ -57,18 +58,18 @@ function App() {
       <MuiThemeProvider theme={theme}>
         <BrowserRouter>
           <Switch>
-
             {/* Routes placed here are only available after logging in and having experience */}
             {user && user.experience.length > 0 && (
               <>
                 <Navbar />
                 <Switch>
                   <Route exact path="/" component={Profile} />
                   <Route exact path="/balance" component={Balance} />
+                  <Route exact path="/balance" component={Balance} />
+                  <Route exact path="/requests/:id" component={Requests} />
```

Comment:
Think we can get rid of the /requests/:id route since the /requests route should handle the routing to the individual reviews. Can you confirm that this works?

Reply:
I tried removing it, but it seems like if we remove it you get redirected back to the homepage.